pkgs/development/misc/avr-gcc-with-avr-libc/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation {
   };
 
   srcAVRLibc = fetchurl {
-    url = http://download.savannah.gnu.org/releases/avr-libc/avr-libc-1.7.1.tar.bz2;
+    url = http://ftp.igh.cnrs.fr/pub/nongnu/avr-libc/old-releases/avr-libc-1.7.1.tar.bz2;
```

Comment:
Ok.
I confirm yesterday, my test was ko because I did not test the right url.
Here is the good one: http://download.savannah.gnu.org/releases/avr-libc/old-releases/avr-libc-1.7.1.tar.bz2